lib/openstudio-standards/standards/Standards.Model.rb

```diff
@@ -1,4 +1,5 @@
 require 'csv'
+require 'date'
```

Comment:
is this in the core or std library?

Reply:
I think this comes from Ruby standard library - see [Ruby 2.7.2](https://ruby-doc.org/stdlib-2.7.2/libdoc/date/rdoc/Date.html)